src/components/ApiHistoryTable.tsx

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+import { Details } from '@docusaurus/theme-common/Details';
+import {
+  ApiHistory,
+  PrReleaseVersions,
+  PrReleaseVersionsContainer,
+} from '../transformers/api-history';
+import ReactMarkdown from 'react-markdown';
+
+import styles from './ApiHistoryTable.module.scss';
```

Comment:
```suggestion
import { Details } from '@docusaurus/theme-common/Details';
import React from 'react';
import ReactMarkdown from 'react-markdown';

import {
  ApiHistory,
  PrReleaseVersions,
  PrReleaseVersionsContainer,
} from '../transformers/api-history';
import styles from './ApiHistoryTable.module.scss';
```

Sorting imports is fun.